coalib/parsing/ConfParser.py

```diff
@@ -48,8 +48,11 @@ def parse(self, input_data, overwrite=False):
         if os.path.isdir(input_data):
             input_data = os.path.join(input_data, Constants.default_coafile)
 
-        with open(input_data, 'r', encoding='utf-8') as _file:
-            lines = _file.readlines()
+        try:
+            with open(input_data, 'r', encoding='utf-8') as _file:
+                lines = _file.readlines()
+        except EnvironmentError as e:
+            print("An error occurred: ", e)
```

Comment:
Also, have you tried to reproduce the error? If the configuration files is not loaded here but the error is already handled, does coala run normally?